src/main/java/org/apache/sysds/runtime/compress/estim/CompressedSizeInfo.java

```diff
@@ -95,6 +95,8 @@ public String getEstimatedDistinct() {
 		StringBuilder sb = new StringBuilder();
 		if(compressionInfo == null)
 			return "";
+		if(compressionInfo.size()<=0)
```

Comment:
if this happened it is a bug. 
we should probably throw an exception

Reply:
Thank you for the comments Sebastian, I've added the requested changes.